src/components/interaction-designer/Block.vue

```diff
@@ -136,6 +140,9 @@ import { ResourceResolver, SupportedMode } from '@floip/flow-runner'
 import { OperationKind, generateConnectionLayoutKeyFor } from '@/store/builder'
 import Connection from '@/components/interaction-designer/Connection.vue'
 import lang from '@/lib/filters/lang'
+import {VBTooltipPlugin} from 'bootstrap-vue'
+
+Vue.use(VBTooltipPlugin)
```

Comment:
Minor thing: I would like to understand if there a specific reason why we use a bootstrap-vue plugin inside a specific component? 
Can we do something like this inside main.ts instead?
```
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue)
```
This would make all the bootstrap components available across the app.

Reply:
@poojahpatil90 , for this particular scenario there is no specific reason. I see it could be confusing, so I changed it to
```
import { BTooltip } from 'bootstrap-vue'

Vue.component('b-tooltip', BTooltip)
```
I want to avoid importing all bootstrap-vue components without using them

Reply:
Thanks for clarifying. Just wanted to understand the reasoning.